simpleflow/swf/executor.py

```diff
@@ -67,9 +67,10 @@ def run_fake_child_workflow_task(domain, task_list, workflow_type_name,
                                  workflow_type_version, workflow_id, input=None, result=None,
                                  child_policy=None, control=None, tag_list=None):
     conn = ConnectedSWFObject().connection
-    conn.start_child_workflow_execution(
-        workflow_type_name, workflow_type_version, workflow_id,
+    conn.start_workflow_execution(
```

Comment:
Ah! You're totally right, there's no action to be taken for the child workflow to start, it's started once the decision is sent and processed by SWF. So actually this function call can should be removed. Thanks!